app/src/main/java/woowacourse/movie/movie/fragment/HistoryFragment.kt

```diff
@@ -0,0 +1,55 @@
+package woowacourse.movie.movie.fragment
+
+import android.content.Intent
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.fragment.app.Fragment
+import androidx.recyclerview.widget.RecyclerView
+import woowacourse.movie.R
+import woowacourse.movie.movie.activity.TicketActivity
+import woowacourse.movie.movie.dto.BookingHistoryDto
+import woowacourse.movie.movie.dto.movie.BookingMovieDto
+import woowacourse.movie.movie.history.HistoryRVAdapter
+import woowacourse.movie.movie.movielist.OnClickListener
+
+class HistoryFragment : Fragment() {
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+    }
+
+    override fun onCreateView(
+        inflater: LayoutInflater,
+        container: ViewGroup?,
+        savedInstanceState: Bundle?,
+    ): View? {
+        // Inflate the layout for this fragment
+        val view = inflater.inflate(R.layout.fragment_history, container, false)
+        setUpHistoryDatas(view)
+
+        return view
+    }
+
+    private fun setUpHistoryDatas(view: View) {
+        val history_rv = view.findViewById<RecyclerView>(R.id.history_rv)
```

Comment:
```suggestion
        val historyRV = view.findViewById<RecyclerView>(R.id.history_rv)
```
코틀린에서는 snackcase 보다는 camelcase를 지향합니다.